### PR TITLE
Update readme about the keyboard shortcut documentation

### PR DESCRIPTION
**Pull request by zhengbli**
Related to #439

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ The below features are available via the keyboard shortcuts shown, or via the Co
 |Go to definition       | `^T^D` or `F12` |
 |Trigger completion     | `^Space`        |
 |Trigger signature help | (Win)`Alt+,`    |
+|See last signature in the tooltip | `Alt + up`   |
```
Comment (sandersn) [inline README.md]: Should this be 'previous' ?

Comment (zhengbli) [inline README.md]: Right, done

File: README.md [hunk 2]
```diff
@@ -61,6 +61,8 @@ The below features are available via the keyboard shortcuts shown, or via the Co
 |Go to definition       | `^T^D` or `F12` |
 |Trigger completion     | `^Space`        |
 |Trigger signature help | (Win)`Alt+,`    |
```
Comment (billti) [inline README.md]: Why does this line say "(Win)"? I don't think this is Windows specific is it?

**Comment by billti** [comment]
One minor comment, else :+1:

**Comment by sandersn** [comment]
:+1:

**Pull request merged**